jfr-static-image-action/action.yml

```diff
@@ -16,11 +16,42 @@ inputs:
   jcasc:
     description: Jenkins CasC file
     required: false    
+  uploadLog:
+    description: whether to upload the pipeline result
+    type: boolean
+    required: false
+    default: false
+  cacheMaven:
+    description: whether to cache maven dependencies
+    type: boolean
+    required: false
+    default: false    
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
-  args:
-    - ${{inputs.command}}
-    - ${{inputs.jenkinsfile}}
-    - ${{inputs.pluginstxt}}
-    - ${{inputs.jcasc}}
\ No newline at end of file
+  using: "composite"
+  steps:
+    - name: Set up cache and workspace
+      run: mkdir -p jenkinsHome jenkins_new_plugins /home/runner/.m2/repository
+      shell: bash
+    - uses: actions/cache@v3
+      id: cache-jenkins
+      name: Cache Jenkins
+      with:
+        path: jenkins_new_plugins
+        key: ${{ runner.os }}-plugins-${{ hashFiles(inputs.pluginstxt) }}
+    - name: Cache Maven
```

Comment:
@oleg-nenashev what would you suggest regarding caching maven dependencies across different runs of the action?